metricflow-semantics/metricflow_semantics/model/semantics/linkable_element.py

```diff
@@ -75,7 +76,12 @@ def __post_init__(self) -> None:
             assert (
                 self.time_granularity
             ), "Time granularity must be specified for all ElementPathKeys associated with time dimensions!"
-        elif element_type is LinkableElementType.DIMENSION or LinkableElementType.ENTITY or LinkableElementType.METRIC:
+        elif element_type is LinkableElementType.METRIC:
```

Comment:
This was mainly a runtime guard in case you forget to add it! Definitely don't need to keep it as is, I think it was probably most useful in making sure I didn't miss any places I needed to add it for this initial implementation 🙃

Reply:
Fair enough, the choice is yours - you'll have a better idea of which is more valuable than I do.

Reply:
I removed it!